packages/store/src/database/backend.ts

```diff
@@ -25,6 +31,9 @@ export interface DatabaseBackend {
 
 // We persist entities wrapped with extra metadata used for updating snapshots.
 export interface DatabaseBackendEntityRecord<E extends Entity> {
-  lastEventID: EventID; // i.e. last in the order as returned by the backend, following logical time, rather than client-local time
   entity: E;
+
+  // Here, "last" refers to the local logical clock (i.e. sequence number), rather than client-local time.
```

Comment:
Took me a few to understand this comment given the addition of `lastEventTimestampMillis` below. Maybe something like:
```
Here, "last" is determined using the local logical (i.e. sequence number), rather than client-local time.
```